packages/vuetify/src/mixins/scrollable/index.ts

```diff
@@ -43,7 +44,7 @@ export default Vue.extend({
      * enabled or disabled
      */
     canScroll (): boolean {
```

Comment:
it seems to be unnecessary, just use `IN_BROWSER`

Reply:
Not part of the scope of this PR.